### ISSUE TITLE
Fork textAngular for official Meteor support

### ISSUE THREAD

**Issue by Urigo** [netanelgilad/meteor-textAngular#3]

**Comment by netanelgilad** [comment]
@Urigo How do you suggest doing it for the packages?
Add a package.js file with the right angular depedency. But on what user to publish those packages? the package owners need to create users for them to it themselves. Do we have a tutorial for how to do it? something we can give them to do it by?

**Comment by Urigo** [comment]
Here is one example:
https://github.com/angular-ui/angular-google-maps/pull/1174

You can publish on your name and then after he registers, publish under his and delete yours.

Here is another example, this time with adding scripts to the build process:
https://github.com/angular-ui/ui-router/pull/1751

you might find more examples here:
https://github.com/MeteorPackaging

**Comment by vimes1984** [comment]
Sorry to highjack this thread, but is this package ready for use as a dependency in another package? 
if so which one am I using: 
fraywing:text-angular 
or
netanelgilad:text-angular
??

**Comment by netanelgilad** [comment]
Hey @vimes1984,
This package should be good for using as a dependency. I don't know about the other one. Have you managed to solve your problem?

**Comment by vimes1984** [comment]
yeah I solved the problem :D thanks for getting back to me!

**Comment by netanelgilad** [comment]
Yeah sorry it took so long, wasn't around for the last 3 months... :)
